### PR TITLE
Notes/GitHub and markdown

### PR DESCRIPTION
**Pull request by milodevhh**
# add H2 and text

### CODE REVIEW

**Comment by alper92** [comment]
COOL! LGTM 👍🏻

**Review by KilianBau — COMMENTED** [review]
LGTM 💯

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+## Hallo World
```
Comment (ninaassmann) [inline README.md]: i would choose one language, so hello world. but looks awesome

**Review by ninaassmann — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (milodevhh) [reply README.md]: ah sure, thats for the hint

File: README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+## Hello World
+This is just a small info for you all folks
```
Comment (doemser) [inline README.md:2]: Ich hinterlasse ienen Kommentar

**Review by doemser — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (milodevhh) [reply README.md:2]: thx
;)

**Pull request merged**